UPGRADE_NOTES.md

```diff
@@ -1,4 +1,55 @@
-### Upgrading from 0.14 to 0.15
+## Upgrading from 0.16 to 0.17
+
+- Ensure you've upgraded to crystal 0.30.0
+- Upgrade Lucky CLI (homebrew)
+
+```
+brew update
+brew upgrade crystal-lang # Make sure you're up-to-date. Requires 0.30.0
+brew upgrade lucky
+```
+
+- Upgrade Lucky CLI (Linux)
+
+- Update `.crystal-version` to `0.30.0`
+
+> Remove the existing Lucky binary and follow the Linux
+> instructions in this section
+> https://luckyframework.org/guides/getting-started/installing#on-linux
+
+- Update versions in `shard.yml`
+  - Lucky should be `~> 0.17`
+- Run `shards update`
+
+### General updates
+- Rename: Action rendering method `text` to `plain_text`.
+- Update: use of `number_to_currency` now returns a String instead of writing to the view directly.
+- Rename: `Avram::Repo` to `Avram::Database` in `config/database.cr`.
+
+### Updating queries
+- Rename: `Query.destroy_all` to `Query.truncate`.
```

Comment:
I think this should be `Query#destroy_all` and `Query.truncate`

Reply:
It would be good to add some examples to clarify this. `User.new.destroy_all` would now be `User.truncate`. Worth mentioning it was an instance  method and is now a class method